sql/2020/16_Ecommerce/top_vendors.sql

```diff
@@ -1,26 +1,33 @@
 #standardSQL
 # 13_01: Top ecommerce vendors by device
+# Excluding apps which are not eCommerce platforms/vendors themselves but are used to identify eCommerce sites. These are signals added in Wappalyzer in 2020 to get better idea on % of eCommerce sites but these are not relevant for vendor % market share analysis
 SELECT
   _TABLE_SUFFIX AS client,
   app,
-  COUNT(0) AS freq,
+  COUNT(DISTINCT url) AS freq,
   total,
-  ROUND(COUNT(0) * 100 / total, 2) AS pct
+  COUNT(DISTINCT url) / total AS pct
 FROM
   `httparchive.technologies.2020_08_01_*`
 JOIN
-  (SELECT _TABLE_SUFFIX, COUNT(0) AS total 
-    FROM `httparchive.summary_pages.2020_08_01_*` 
-    GROUP BY _TABLE_SUFFIX)
+  (SELECT 
+      _TABLE_SUFFIX,
+      COUNT(DISTINCT url) AS total
+    FROM 
+      `httparchive.summary_pages.2020_08_01_*` 
+    GROUP BY 
+      _TABLE_SUFFIX)
 USING (_TABLE_SUFFIX)
 WHERE
   category = 'Ecommerce'
   and 
-  app != 'Cart Functionality'
+  (app != 'Cart Functionality' AND 
+   app != 'Google Analytics Enhanced eCommerce')
 GROUP BY
   client,
-  total,
-  app
+  app,
+  total
 ORDER BY
-  freq / total DESC
-LIMIT 100
\ No newline at end of file
+  client desc,
+  pct DESC
+LIMIT 1000
```

Comment:
```suggestion
  client DESC,
  pct DESC
LIMIT 1000
```

BTW Be aware because of your order by clause, this will LIMIT will affect the second set of results (desktop) and not the first (mobile). Not an issue here as only 300 results but might be better removing to void confusion in case that grows (in which case probably should have `HAVING freq > 10` (for some other limit) after `GROUP BY` so you limit both sets of results in the same way.